app/Jobs/NotifyUserOfNewGistComment.php

```diff
@@ -86,4 +87,11 @@ private function isCommentNew()
         return NotifiedComment::where('github_id', $this->comment['id'])
             ->count() == 0;
     }
+
+    public function getUnsubscribeUrl($user)
+    {
+        if ($user) {
+            return $user->getUnsubscribeHash();
```

Comment:
I think this is the wrong method--which brings up that maybe this should be tested?